app/js/controllers/gtmPluginCtrl.js

```diff
@@ -18,7 +18,9 @@ KMCMenu.controller('gtmPluginCtrl', ['$scope',
 		}
 
 		$scope.customListOfCustomEventsChange = function(){
-			$scope.playerData.config.plugins["googleTagManager"].customEventsTracking.custom = $scope.customListOfCustomEvents.split(',');
+			$scope.playerData.config.plugins["googleTagManager"].customEventsTracking.custom = $scope.customListOfCustomEvents
+				.replace(/\s/g,'')
```

Comment:
@JonathanTGold maybe \s+ ?

Reply:
we don't need that, since we made a global search